api/anilist/src/main/kotlin/com/imashnake/animite/api/anilist/AnilistUserRepository.kt

```diff
@@ -14,12 +16,12 @@ import javax.inject.Inject
 class AnilistUserRepository @Inject constructor(
     @AuthorizedClient private val apolloClient: ApolloClient
 ) {
-    fun fetchViewer(): Flow<Result<ViewerQuery.Viewer>> {
+    fun fetchViewer(refresh: Boolean): Flow<Result<ViewerQuery.Viewer>> {
         return apolloClient
             .query(ViewerQuery())
```

Comment:
I guess we couldn't do the observe thing?

Reply:
From what I understand, [`watch()` needs a cache `id`](https://www.apollographql.com/blog/add-graphql-to-your-jetpack-compose-apps#graphql). It's more appropriate to use it for mutations within the app I think.

Reply:
I meant a subscription
https://www.apollographql.com/docs/kotlin/essentials/subscriptions/